src/material/core/focus_indicator/_focus-indicator.scss

```diff
@@ -0,0 +1,61 @@
+@import '../theming/theming';
+
+@mixin mat-strong-focus-indicators() {
+  // Base styles for focus indicators.
+  .mat-focus-indicator::before {
+    border-radius: 4px;
+    border-width: 2px;
+    border-style: solid;
+    box-sizing: border-box;
+    position: absolute;
+    pointer-events: none;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+  }
+
+  // Default insets/outsets for focus indicators.
+  .mat-focus-indicator {
+    &.mat-button-base::before,
+    &.mat-card::before,
+    &.mat-sort-header-button::before {
+      margin: -4px;
+    }
+
+    &.mat-calendar-body-cell::before {
+      margin: -2px;
+    }
+
+    &.mat-tab-link::before,
```

Comment:
I feel like all of these rules should be somewhere next to the styles of the component they're associated with, otherwise we'll end up shipping styles for components that the consumer isn't using.

Reply:
Either we ship focus indicator styles whne users aren't using focus indicators, or we ship component styles when users aren't using particular components. I'm fine either way.

Reply:
I'd prefer to have them all here for now since it's relatively tiny and most people won't turn on the indicators.